lib/challenge.js

```diff
@@ -58,9 +97,25 @@ var some_pet;
 //    the values of `name`, `species`, `breed`, and `noise`. The constructor function should also give each pet a method called `makeNoise`,
 //    which behaves as described in Q3.
 
-var Pet;
 // Answer Starts Here
 
+var Pet = function(name, species, breed, noise){
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = noise;
+  // option 1
+  this.makeNoise = function(){
```

Comment:
Nope - not this way. We don't want to define methods in our Constructor function, because that will cause multiple copies of our functions to be created. By putting methods on the prototype, the only get created once.